brjs-sdk/sdk/libs/javascript/br-presenter/test-unit/src-test/br/presenter/testing/node/RootPresentationNode.js

```diff
@@ -6,6 +6,7 @@ var WritableProperty = require("br/presenter/property/WritableProperty");
 var RootPresentationNode = function()
 {
 	this.m_oPrivateProperty1 = new WritableProperty();
+	this._oAnotherPrivateProperty1 = new WritableProperty();
```

Comment:
It's used by the fact that it doesn't show up as a property in the tests that asset against `properties().length`.